dotnet-docs/windows-installation.md

```diff
@@ -10,48 +10,55 @@ position: 2
 
-The installation wizard will do the initial configuration of _RS.NET_ and _RS.NET Service Agent_ on Windows, making them ready-to-run.
+1. Double-click the downloaded MSI file to start the installer.
+1. Read and accept the license agreement.
+1. Optionally, choose **Customize** to customize the installed options, then click **Next**.
+    >caution The RS.NET installer currently does not include SDK examples. This may be added in future releases.
+1. Choose a user option and click **Next**. Creating a dedicated Windows user (`RSUserNET`) is recommended for security purposes. For more information about the available user options, see [ReportServerUser, LocalSystem Identity and Dedicated Users]({%slug installation%}#reportserveruser-localsystem-identity-and-dedicated-users).
+1. Click **Install** to proceed with the installation.
 
-If the automatic configuration fails, please, get familiar with the [initialization process](#initialization-process) and follow the [instructions for manual configuration](#manual-configuration-on-windows).
+The installer automatically creates IIS applications on port 81 (RS.NET) and port 82 (RS.NET Service Agent). If these ports are unavailable, it assigns the next available ones.
 
-### Initialization process
+When you first access RS.NET, you will be prompted to configure storage and register an administrator user.
 
-1. When the Report Server is started for the first time, the user is supposed to pass the _Configure Storage_ and _Register Administrator_ pages. The settings from these pages are stored in a file named `ReportServerAdmin.json`.
-1. Next, the RS.NET checks its `appsettings.json` configuration file for the key __InitialAgentUrl__. If the installation has passed successfully, the key must exist and must have a valid value like __http://localhost:84__. This is where the MSI installation file for Windows has registered the __RS.NET Service Agent__.
-1. The RS.NET calls the above URL and passes the storage settings to its Service Agent. They are saved in the file `ServiceAgent.json` in the RS.NET Service Agent's directory. If such a file does not exist, the agent was not initialized or registered in the IIS.
+## Upgrading Report Server for .NET
 
-### Manual Configuration on Windows
+To upgrade RS.NET from a previous version, download and run a new installer file and follow the [installation steps](#installing-report-server-for-net).
 
-1. Delete the file `\Services\.NET\ServiceAgent.json` from RS.NET Service Agent's folder if it exists.
-1. Test whether the RS.NET Service Agent responds by calling the RS.NET Service Agent endpoint `/api/system/isalive` from the browser. By default, this would be the URL `http://localhost:84/api/system/isalive`.
+When upgrading from the combined MSI installer (versions prior to 2025 Q4), environment variables (such as encryption keys) are automatically migrated to the newly selected user option (for example, `RSUserNET` if creating a dedicated Windows user).
```

Comment:
Along with the environment variables, we will migrate also:
1) the storage configuration (the info where the RS storage assets are kept);
2) the storage itself, in case the old RS installation have used FileStorage.
The old assets location created by the combined installer will not be deleted by the migration logic.